src/main/protractor/Tests/profileSettings.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Created by juan muñoz on 30/06/17.
+ */
+
+describe('Profile settings testing Web App', function() {
+
+    beforeAll(function() {
```

Comment:
we need to add before.each and before.all, in before.all function only should be for the url